monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -102,39 +113,56 @@ def get_tunnels():
     def get_scanned():
         formatted_nodes = []
 
-        nodes = ReportService.get_all_displayed_nodes()
+        machines = ReportService.get_all_machines()
 
-        for node in nodes:
+        for machine in machines:
             # This information should be evident from the map, not sure a table/list is a good way
             # to display it anyways
-            nodes_that_can_access_current_node = node["accessible_from_nodes_hostnames"]
+            addresses = [str(iface.ip) for iface in machine.network_interfaces]
```

Comment:
why do we need `str(...)`? Won't jsonifying turn it into string?

Reply:
Looks like the answer is no: `TypeError: Object of type IPv4Address is not JSON serializable`